DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateGroups/EditGroupNameViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using System.ComponentModel;
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Web.Helpers;
+
+    public class EditGroupNameViewModel
+    {
+        public EditGroupNameViewModel() { }
+
+        public EditGroupNameViewModel(string groupName)
+        {
+            GroupName = groupName;
+        }
+
+        [DisplayName("Group name")]

```

Comment:
Does this do anything?

Reply:
As we have a generic constant StringMaxLengthValidation = "{0} must be {1} characters or fewer", this produces the below generic validation and can be reused. {0} takes [DisplayName("Group name")] and {1} is length 

![image](https://user-images.githubusercontent.com/90783892/143403016-f156acc4-e984-4363-8535-92841e6b1a47.png)